validateSql.hs

```diff
@@ -1,33 +1,36 @@
 #!/usr/bin/env stack
```

Comment:
NOTE this should "call out" to another `Lib.hs` style package for maximum portability, can probably be avoided for now. If not refactored, keep the shebang for backwards compatibility & document.